test/plugins/navigation/review/navigator/test.js

```diff
@@ -488,7 +488,36 @@ define([
             );
             assert.equal(
                 $('.fizzy > .qti-navigator-label > .qti-navigator-text', $container).text(),
-                'Test review',
+                'Test overview',
+                'Review panel title still exists'
+            );
+
+            ready();
+        })
+        .render($container)
+        .update(sample.map, sample.context);
+    });
+
+    QUnit.test('check open button flow', function(assert) {
```

Comment:
Isn't this test a copy of `QUnit.test('check disabling section titles', function(assert) {` but with undefined value for `displaySectionTitles`? So basically what you are checking is that default value for this option is `false`. If I'm right, maybe rename this test case to e.g."_check default for section titles option_"?

Reply:
This was try to check close/open functionality. Unfortunatley QUnit do not have method like in Svelte `tick()`. I tried to find a way to wait tile panel was updated finally + find method where i can call `ready()`, but didn't find this. As of complexity of such investigation much more, than profit of the test, so i remove it. May be after finalizing my tasks i can return to this problem.